molgenis-data-row-edit/src/main/frontend/src/components/DataRowEdit.vue

```diff
@@ -15,7 +15,7 @@
 
     <div v-if="showForm">
 
-      <h5>{{dataTableLabel}}</h5>
+      <h1>{{dataTableLabel}}</h1>
```

Comment:
This seems strange to me as this is the label of a form, a suspect we should reserve h1 for main page headers ?
maybe jou could add screen shot to make clear the issue this PR is trying to solve

Reply:
i see what you mean, but i'm thinking why is the 'xcomound label' that big ..